tests/test_cli.py

```diff
@@ -0,0 +1,52 @@
+import os
+import subprocess
+
+
+def test_backup_restore():
+    """Backup and restore works"""
+
+    subprocess.call(
+        [
+            "docker", "run", "-d", "--rm",
```

Comment:
I think you can call this prior to running the tests, in the .travis file for example. That way you can debug Docker separately from the tests themselves.

Reply:
I'll need to kill the container to get a new database for restore. This is why I didn't want to assume the container is running.

Reply:
Probably better to wipe it instead.